src/eureka/S3_data_reduction/s3_reduce.py

```diff
@@ -111,6 +111,19 @@ def reduce(eventlabel, ecf_path=None, s2_meta=None, input_meta=None):
     else:
         meta = me.mergeevents(meta, s2_meta)
 
+    # Increase relevant meta parameter values
+    if not hasattr(meta, 'expand'):
+        meta.expand = 1
+    if meta.expand > 1:
+        meta.spec_hw *= meta.expand
+        meta.bg_hw *= meta.expand
```

Comment:
Can you confirm that this does not have any unintended consequences on filenames or foldernames?

Reply:
Ah, I can see from your testing edits that it does change the filenames. I really don't think that's a good idea - ap8 with spec_hw=8 & expand=1 is very different from ap8 with spec_hw=4 & expand=2. I think it's much better to leave the filenames unchanged

Reply:
I still think changing the folder names based on the `expand` setting is really confusing. I think it's worth just dividing by `meta.expand` in the few rare cases where we need the the input `spec_hw` and `bg_hw`. I think the one area where we define the output folder and Fig 3304 are the only places where we'd need to divide by `meta.expand`

Reply:
Ok, I've made a few more edits in this section to:
- Prevent using expand with photometry
- Work correctly when spec_hw or bg_hw is a list
- Use folder names that divide by meta.expand